Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/DecomposeRelationshipTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Relationships.Domain.Aggregates.Relationships;
+using Backbone.Modules.Relationships.Domain.Tests.Extensions;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+using static Backbone.Modules.Relationships.Domain.Tests.TestHelpers.TestData;
+
+namespace Backbone.Modules.Relationships.Domain.Tests.Tests.Aggregates.Relationships;
+
+public class DecomposeRelationshipTests
+{
+    [Fact]
+    public void Decomposing_relationship_transitions_relationship_to_status_deletion_proposed()
+    {
+        // Arrange
+        var relationship = CreateTerminatedRelationship();
```

Comment:
I'd suggest to make it explicit:

```suggestion
        var relationship = CreateTerminatedRelationship(IDENTITY_1, IDENTITY_2);
```

Same for the other tests

Reply:
Changes made.